src/routes/game/GameView.vue

```diff
@@ -640,7 +645,7 @@ export default {
       return this.selectionIndex !== null ? this.gameStore.player.hand[this.selectionIndex] : null;
     },
     turnText() {
-      return this.gameStore.isPlayersTurn ? 'YOUR TURN' : "OPPONENT'S TURN";
+      return this.t(this.gameStore.isPlayersTurn ? 'game.counter.yourTurn' : 'game.counter.opponentTurn');
```

Comment:
Let's move `yourTurn` and `opponentTurn` under a new key `game.turn`. Even though they are visually close to the score info, we have ambitions of moving them further apart when implement a point counter indicator